eclass/nodejs-mod.eclass

```diff
@@ -0,0 +1,110 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: nodejs-mod.eclass
+# @MAINTAINER:
+# Fco. Javier Félix <web@inode64.com>
+# @AUTHOR:
+# Fco. Javier Félix <web@inode64.com>
+# @SUPPORTED_EAPIS: 8
+# @BLURB: An eclass for build NodeJS projects
+# @DESCRIPTION:
+# An eclass providing functions to build NodeJS projects
+#
+# Credits and ideas from:
+#   Initial version from:
+#       https://github.com/gentoo/gentoo/pull/930/files
+#       https://github.com/samuelbernardo/ssnb-overlay/blob/master/eclass/npm.eclass
+#       https://github.com/gentoo-mirror/lanodanOverlay/blob/master/eclass/nodejs.eclass
+#       https://github.com/Tatsh/tatsh-overlay/blob/master/eclass/yarn.eclass
```

Comment:
Is there a reason you haven't given feeback to @Tatsh instead at https://github.com/gentoo/gentoo/pull/30415?

Reply:
I didn't know about the existence of this pull, but I did know about its overlay and I got some ideas from its eclass, but I haven't reviewed it again until now, I don't know if we could somehow merge the 2 systems